content/second-order-logic/syntax-and-semantics/introduction.tex

```diff
@@ -14,8 +14,8 @@
 language, i.e., its !!{constant}s, !!{function}s, and !!{predicate}s,
 with the logical symbols to express things about first-order
 !!{structure}s.  This is done using the notion of satisfaction, which
-relates !!a{structure}~$\Struct{M}$, together with a variable
```

Comment:
This is actually the way it's supposed to be. !!a inserts the proper article (eg if you redefine structure to print as interpretation, you will get "an interpretation" everywhere.

I can accept the pull request and undo this change (then you have to pull that change), or you can undo it and revise the pull request (git revert 301d9feb068653893c044903d2850a151fc8008d) .... whatever is easier for you.